pkg/workflows/wasm/host/module.go

```diff
@@ -57,13 +57,14 @@ func (r *store[T]) add(id string, req *RequestData[T]) error {
 	return nil
 }
 
-func (r *store[T]) get(id string) (*RequestData[T], error) {
+func (r *store[T]) get(id string) (T, error) {
```

Comment:
not a fan of using error to indicate not found, better to follow map semantics and return a bool